khmer/khmer_args.py

```diff
@@ -389,18 +393,22 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0, _sysmem=None):
```

Comment:
Why have the `_sysmem` argument? Is it solely to make the unittests work? In python you can use [mock](https://docs.python.org/3/library/unittest.mock.html) to replace the `psutil` call for the unit test. Unfortunately it isn't part of the stdlib for `python2`.

(Function arguments starting with a `_` attract attention in python land 👀